### ISSUE TITLE
A11y_Visual Studio Code Services_Docs Python_Testing_Use Of Color: Links text on the page are not associated with underline/chevron to differentiate from normal text on page.

### ISSUE THREAD

**Issue by manaswita-chougule** [microsoft/vscode-docs#6321]
### GitHub Tags
#WCAG1.4.1; #A11ySev3; #Use Of Color; #A11yTCS; #A11yWCAG2.1; #Chrome; #DesktopWeb; #Win11; #Visual Studio Code Services; #BM_VisualStudioCodeServices_Web_Apr2023;

### Environment Details
Application Name: Visual Studio Code Services
URL: [https://code.visualstudio.com/](https://code.visualstudio.com/)
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Chrome Version 113.0.5672.93 (Official Build) (64-bit)

### Repro Steps

1. Hit the application URL: https://code.visualstudio.com/
2. Tab till "Docs" link and press enter.
3. Tab till "Python" link in the left navigation and press enter.
4. Tab till "Testing" link which is present under the python and press enter.
5. Observe that Link text on the page are not associated with underline/chevron to differentiate from normal text on page.

### Actual Result
Link text on the page are not associated with underline/chevron to differentiate from normal text on page.
Note:
Similar issue is observed throughout the application for link texts and for on-page heading links as well.

### Expected Result
Link text on the page should be associated with underline/chevron to differentiate from normal text on page.

### User Impact
Color-blind users will not be able to distinguish between normal text and link text on the page, so they face difficulties to access the links easily.

### Attachment:
![Use of color](https://github.com/microsoft/vscode-docs/assets/91586101/8ae17cd9-9685-48cb-9bbb-0b18f05fa8f1)

**Comment by rzhao271** [comment]
I believe this issue is a duplicate of https://github.com/microsoft/vscode-docs/issues/6057. That issue states the following:

```
How to fix:
Fix **any** of the following:
The link has insufficient color contrast of 1.88:1 with the surrounding text. (Minimum contrast is 3:1, link text: #0098ff, surrounding text: #cccccc)
The link has no styling (such as underline) to distinguish it from the surrounding text
```

The colour contrast of the links should be at least 3:1 now, thus meeting the requirements above.

**Comment by rzhao271** [comment]
Thanks! Closing as a duplicate.
